src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,47 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("completeOrder method was called for userId: {}", userId);
+        List<Product> products = null;
+        try {
+            products = getAllProductsFromShoppingCart(userId);
+            logger.debug("Products retrieved from shopping cart for userId: {}: {}",
```

Comment:
While the log message includes user ID and products, it's only a debug log. Consider adding more informative logging before the try block to indicate what operation is being performed, such as retrieving products from the shopping cart.